R/caret.R

```diff
@@ -4,10 +4,10 @@
 #'  \pkg{rsample} and \pkg{caret}.
 #'
 #' @param object An `rset` object. Currently,
-#'  `nested_cv` is not supported.
-#' @return `rsample2caret` returns a list that mimics the
+#'  `nested_cv()` is not supported.
+#' @return `rsample2caret()` returns a list that mimics the
 #'  `index` and `indexOut` elements of a
-#'  `trainControl` object. `caret2rsample` returns an
+#'  `trainControl` object. `caret2rsample()` returns an
```

Comment:
For external functions, you can namespace them

```suggestion
#'  `[caret::trainControl()]` object. [caret2rsample()] returns an
```

Reply:
We'll trigger a CRAN note when cross-referencing functions from packages we don't depend on so let's keep references to such packages (like caret here) in backticks instead of switching to the brackets.